data/deep missions.txt

```diff
@@ -1122,32 +1131,32 @@ mission "Deep: Project Hawking: Carbuncle"
 			`	"Project Hawking?" you ask. Garrison nods.`
 			
 			label project
-			`	"The exact contents of Project Hawking are classified. I'm not able to tell you much aside from the fact that we're doing energy and weapons research. Scientists from the Deep who are highly specialized in a certain field will be asked to work on government projects every so often.`
+			`	"The exact nature of Project Hawking is classified. I'm not able to tell you much aside from the fact that we're doing energy and weapons research. Scientists from the Deep who are highly specialized in a certain field will be asked to work on government projects every so often.`
```

Comment:
Would the nature not be that it relates to energy and weapons research, meaning that Garrison just told you classified information? I feel that `contents` would be a better word than `nature` because of this.

Reply:
"details"?

Reply:
Sure.